corehq/util/datadog/metrics.py

```diff
@@ -1,5 +1,5 @@
 JSERROR_COUNT = 'jserror.count'
 ERROR_COUNT = 'error.count'
-REPEATER_ERROR_COUNT = 'repeater.error.count'
+REPEATER_ERROR_COUNT = 'commcare.repeater.error.count'
```

Comment:
Also I think error_count may be redundant. could probably just be commcare.repeaters.error